### ISSUE TITLE
Create a Subscribe form 

### ISSUE THREAD

**Issue by sau-mili** [Its-Aman-Yadav/Community-Site#28]

**Comment by Pooja-c03** [comment]
i would love to make this form, please assign this to me!

**Comment by FatemaBaig** [comment]
Hello,
i would like to work on this part of project. Please assign it to me.

**Comment by lodhivishalcoder28** [comment]
@Its-Aman-Yadav i can make the form , please assign this task to me

**Comment by Its-Aman-Yadav** [comment]
what will be the use of a subscribe button here?

**Comment by sau-mili** [comment]
> what will be the use of a subscribe button here?

for collecting e-mail ids to send website updates to subscribers

**Comment by Its-Aman-Yadav** [comment]
I don't think so there is a need of a subscribe button right now but we can give it a try, please work on it @sau-mili

**Comment by sau-mili** [comment]
### 
Check my Pull Request
- [#239  ]

**Comment by sau-mili** [comment]
#239

**Comment by sau-mili** [comment]
#240

**Comment by sau-mili** [comment]
[#311 ](url) please check

**Comment by harshcodesit** [comment]
My name is Harshwardhan Singh Panwar and I'm a contributor at GSSOC'24 .

idea:- By incorporating div CSS and animations, I can design an appealing subscribe form

Please assign this task to me, GSSOC'24 .

**Comment by DiwareNamrata23** [comment]
@sau-mili I'm interested for contributing into this form . I request you to assign this task to me
